### PR TITLE
Metadata

### PR DESCRIPTION
**Pull request by fabientownsend**

### CODE REVIEW

File: spec/server_spec.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@
 require 'rspec'
 require 'server'
 require 'spec_helper'
+require 'web_player'
```
Comment (christophgockel) [inline spec/server_spec.rb:6]: What's that for in the spec?

File: config.ru [hunk 2]
```diff
@@ -0,0 +1,9 @@
+$: << File.expand_path('./lib', File.dirname(__FILE__))
+
+require 'rack'
+require 'server'
+
+Rack::Handler::WEBrick.run(
```
Comment (christophgockel) [inline config.ru:6]: Nice to see a rackup file! 👍

File: lib/server.rb [hunk 3]
```diff
@@ -3,48 +3,76 @@
...
-    if (env["PATH_INFO"] == "/" || env["PATH_INFO"] == "/reset")
-      start_new_game
-      html = HTMLBuilder.generate_page("Start game", HTMLBuilder.board(@board.board))
-    elsif (env["PATH_INFO"] == "/move")
-      @game.play(position(env))
+    path = env["PATH_INFO"]
+    @env = env
+
+    if (path == "/" || path == "/reset")
+      initialize(@vs_computer)
```
Comment (christophgockel) [inline lib/server.rb:33]: Calling `initialize` directly is not idiomatic ruby as it is the constructor of your class.

File: spec/server_spec.rb (hunk 1 above)
Comment (fabientownsend) [reply spec/server_spec.rb:6]: dead code. It should be there!

File: lib/server.rb (hunk 3 above)
Comment (fabientownsend) [reply lib/server.rb:33]: I'm wonder it a GameBuilder would  make sense. The I could call it from the initializer  and other method that need it.

File: spec/server_spec.rb (hunk 1 above)
Comment (christophgockel) [reply spec/server_spec.rb:6]: > dead code. It should be there!

I assume you mean it _shouldn't_? ;)

**Pull request closed without merging**